Content.Server/Disposal/Unit/EntitySystems/DisposalUnitSystem.cs

```diff
@@ -468,9 +470,17 @@ private bool Update(DisposalUnitComponent component, float frameTime)
 
         public bool TryInsert(EntityUid unitId, EntityUid toInsertId, EntityUid userId, DisposalUnitComponent? unit = null)
         {
+            var popupSystem = EntitySystem.Get<SharedPopupSystem>();
```

Comment:
just add a dependency at the top of the file rather than getting the EntitySystem here?

Reply:
Just added the dependency!